### DISCUSSION TITLE
CLI: initialise a new project with pages router without prompt

### DISCUSSION THREAD

**Discussion by mishushakov** [vercel/next.js#69697] [Help]
### Link to the code that reproduces this issue

https://github.com/mishushakov/create-next-issue

### To Reproduce

Run:

```
npx create-next-app@latest . --ts --tailwind --no-eslint --import-alias "@/*" --use-npm --no-src-dir
```

### Current vs. Expected behavior

Current:

Will ask me explicitly:

```
? Would you like to use App Router? (recommended) › No / Yes
```

Expected:

Proceeds to create an app using pages router, no questions asked

### Provide environment information

```bash
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 23.5.0: Wed May  1 20:12:58 PDT 2024; root:xnu-10063.121.3~5/RELEASE_ARM64_T6000
  Available memory (MB): 16384
  Available CPU cores: 8
Binaries:
  Node: 20.12.2
  npm: 10.5.0
  Yarn: 1.22.21
  pnpm: 8.15.4
Relevant Packages:
  next: 14.2.4 // There is a newer version (14.2.6) available, upgrade recommended! 
  eslint-config-next: 14.2.4
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.5.4
Next.js Config:
  output: N/A
 ⚠ There is a newer version (14.2.6) available, upgrade recommended! 
   Please try the latest canary version (`npm install next@canary`) to confirm the issue still exists before creating a new issue.
   Read more - https://nextjs.org/docs/messages/opening-an-issue
```

### Which area(s) are affected? (Select all that apply)

create-next-app

### Which stage(s) are affected? (Select all that apply)

next dev (local)

**Comment by caoimhebyrne** [comment]
Is the question you're asking related to the fact that there is no CLI option for the pages router, or the fact that it should default to it 🤔

**Comment by mishushakov** [comment]
I expect it to default to pages router, yes

**Comment by caoimhebyrne** [comment]
Why should it default to the pages router..? App router is recommended these days, nothing you're supplying in the CLI arguments indicates that it should use the pages router by default.

**Comment by samcx** [comment]
@mishushakov You can do this via the `--no-app` option.
